.github/workflows/pr-unit-test.yml

```diff
@@ -23,31 +23,31 @@ jobs:
       matrix:
         include:
           - os: el8
-            image: latest
+            image: d9f98e8
             build:
               type: Release
           - os: ubuntu-focal-deb
-            image: latest
+            image: d9f98e8
             build:
               type: Release
-          - os: ubuntu-focal
-            image: latest
+          - os: ubuntu-jammy
+            image: d9f98e8
             build:
               type: Release
           - os: el8
-            image: latest
+            image: d9f98e8
             build:
               type: LCheck
           - os: el8
-            image: latest
+            image: d9f98e8
             build:
               type: TCheck
           - os: el8
-            image: latest
+            image: d9f98e8
             build:
               type: DCheck
           - os: el8
-            image: latest
+            image: d9f98e8
```

Comment:
Change this to specific images so when we build new image does not affect the current master and if we update some library that requires code changes, does not break previous commit builds